### ISSUE TITLE
deceptive site workaround

### ISSUE THREAD

**Issue by mahnouman** [filetrust/k8-reverse-proxy#21]
As a byproduct of the functionality of **reverse proxy** with a **rewritten URL** which is **publicly accessible** (This combination is important); Google and others marked the reverse proxies website as a deceptive one:
![deceptive-site-warning](https://user-images.githubusercontent.com/5932628/92573322-eaad0080-f285-11ea-8ac6-7f08b27a6fa6.png)
Moreover, the hosting provider was also warned about this, please propose a suitable solution / workaround for this.

**Comment by JOSHUAJEBARAJ** [comment]
We can go got review https://developers.google.com/web/fundamentals/security/hacked/request_review

**Comment by michaelhidalgo** [comment]
It seems like the issue is two-fold:

- On one hand, browsers (Firefox, Chrome) mark the site as deceptive. I have been playing with Safari and it works fine, however, I think is just a matter of time for it to flag it.
- AWS also struggling with the URL flagged as phishing.

I'm investigating if by having a wildcard certificated for *.glasswall-icap.com should avoid this situation. Because multiple browser vendors will have different ways for detecting phishing.

**Comment by michaelhidalgo** [comment]
It seems like the issue is two-fold:

- On one hand, browsers (Firefox, Chrome) mark the site as deceptive. I have been playing with Safari and it works fine, however, I think is just a matter of time for it to flag it.
- AWS also struggling with the URL flagged as phishing.

I'm investigating if by having a wildcard certificated for *.glasswall-icap.com should avoid this situation. Because multiple browser vendors will have different ways for detecting phishing.

**Comment by michaelhidalgo** [comment]
Yeah, but the problem is that it is now on the blacklist from the browsers (aka google and firefox) so we need to get back to them individually. However, if we deploy another for example for owasp, something like owasp.glasswall-icap.com will be picked up as well.

Yeah, that's right about the TLD for ..,  let me see if the Basic Auth protects it somehow.

**Comment by michaelhidalgo** [comment]
I started a conversation with them just to understand if putting the sites behind a login wall will stop the issue from happening, but at the moment they need to whitelist the one for gov.uk

**Comment by michaelhidalgo** [comment]
So here are some links of interests:

- Really great article from Google https://developers.google.com/safe-browsing/v4/advisory

- Also the AntiPhishing org https://apwg.org/
